api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/IssueTrackerChannel.java

```diff
@@ -30,8 +31,8 @@ protected IssueTrackerChannel(IssueTrackerProcessorFactory<D, T> processorFactor
     }
 
     @Override
-    public MessageResult distributeMessages(D distributionDetails, ProviderMessageHolder messages) throws AlertException {
-        IssueTrackerProcessor<T> processor = processorFactory.createProcessor(distributionDetails);
+    public MessageResult distributeMessages(NamedDistributionJobDetailsModel<D> distributionDetails, ProviderMessageHolder messages) throws AlertException {
+        IssueTrackerProcessor<T> processor = processorFactory.createProcessor(distributionDetails.getDistributionJobDetailsModel());
```

Comment:
Do people not want this on issue-tracker channels? Maybe included with the _create_ comment?

Reply:
They do, haven't looked into the best place to make this change for issue trackers but will probably fix this in my next PR which actually adds the feature we want.